components/consentRequestForm/dateInput/index.test.jsx

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright 2020 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import React from "react";
+import userEvent from "@testing-library/user-event";
+import DateFnsUtils from "@date-io/date-fns";
+import { MuiPickersUtilsProvider } from "@material-ui/pickers";
+import { renderWithTheme } from "../../../__testUtils/withTheme";
+import { ConsentRequestProvider } from "../../../src/contexts/consentRequestContext";
+import ConsentRequestFrom from "../index";
+import {
+  TESTCAFE_ID_FOREVER_BUTTON,
+  TESTCAFE_ID_DATE_PICKER_CALENDAR_BUTTON,
+} from "./index";
+
+describe("DateInput component", () => {
```

Comment:
Is this different somehow from the request we get from `getRequestDetails`?

Reply:
oh that's a miss from my side. It was just a simplified one with one single section as I wrote it. Was meant to update it to the mock one. I'll make this change